src/store_client.cc

```diff
@@ -10,6 +10,7 @@
 
 #include "squid.h"
 #include "acl/FilledChecklist.h"
+#include "AccessLogEntry.h"
```

Comment:
We need to keep them in alphabetical order.

Reply:
Fixed in commit 9adb23e.